README.md

```diff
@@ -16,6 +16,36 @@
 pip install nebari-plugin-self-registration
 ```
 
+## User Registration via this extension
+Steps for self registration:
+
+- Navigate to your Nebari domain.
+<p align="center">
+  <img src="images/welcome-nebari.png" />
+</p>
+
+- Click the link at "To register for an account, click here." or go to https://{your-domain-name}/registration
```

Comment:
This doesn't match the image above. The image needs to have this text.

Reply:
Good catch, the image that had that hyper link had JATIC specific language we were trying to remove. I changed it up a bit, but maybe it just makes the most sense for step 1 to be "Navigate to https://{your-domain-here}/registration"

Reply:
Resolved by: https://github.com/MetroStar/nebari-self-registration/pull/9/commits/05fba93705caaef8d617b2635558592a349c5951